src/monster.h

```diff
@@ -429,6 +429,7 @@ class monster : public Creature
         }
 
         short ignoring;
+        int lastseen_turn = INT_MIN;
```

Comment:
Please use `time_point` instead of `int` for time points. And as it's value is optional, wrap it into a `cata::optional` instead of using an arbitrary value as sentinel.

Reply:
Got it. Pushing changes now.